Documentation/design/dependency-resolution.md

```diff
@@ -1,6 +1,6 @@
 # Dependency Resolution and Upgrades
 
-OLM manages the dependency resolution and upgrade lifecycle of running operators. In many ways, thes problems OLM faces are similar to other OS pacakge managers like `apt`/`dkpg` and `yum`/`rpm`.
+OLM manages the dependency resolution and upgrade lifecycle of running operators. In many ways, these problems OLM faces are similar to other OS pacakge managers like `apt`/`dkpg` and `yum`/`rpm`.
```

Comment:
> In many ways, these problems OLM faces are similar to other OS pacakge managers like `apt`/`dkpg` and `yum`/`rpm`.

In many ways, the problems OLM faces are are similar to those faced by other OS package managers like `apt`/`dpkg` and `yum`/`rpm`.